platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/NonAncientEventWindow.java

```diff
@@ -37,112 +38,54 @@ public class NonAncientEventWindow {
-            final long minGenNonExpired,
-            final long roundsNonAncient,
-            final long roundsNonExpired,
-            @NonNull final AncientMode ancientMode) {
-
-        return new NonAncientEventWindow(
-                latestConsensusRound,
-                latestConsensusRound - roundsNonAncient + 1,
-                minGenNonAncient,
-                minGenNonExpired,
-                latestConsensusRound - roundsNonExpired + 1,
-                ancientMode);
-    }
-
-    /**
-     * Construct using settings in the provided context.
-     *
-     * @param latestConsensusRound the latest round that has come to consensus
-     * @param minGenNonAncient     the minimum generation that is non-ancient
-     * @param minGenNonExpired     the minimum generation that is non-expired
-     * @param platformContext      the platform context
-     * @return the new NonAncientEventWindow
-     */
-    @NonNull
-    public static NonAncientEventWindow createUsingPlatformContext(
-            final long latestConsensusRound,
-            final long minGenNonAncient,
-            final long minGenNonExpired,
-            @NonNull final PlatformContext platformContext) {
-
-        final ConsensusConfig consensusConfig =
-                platformContext.getConfiguration().getConfigData(ConsensusConfig.class);
-
-        final long roundsNonAncient = consensusConfig.roundsNonAncient();
-        final long roundsNonExpired = consensusConfig.roundsExpired();
-
-        final AncientMode ancientMode = platformContext
-                .getConfiguration()
-                .getConfigData(EventConfig.class)
-                .getAncientMode();
-
-        return createUsingRoundsNonAncient(
-                latestConsensusRound,
-                minGenNonAncient,
-                minGenNonExpired,
-                roundsNonAncient,
-                roundsNonExpired,
-                ancientMode);
-    }
-
     /**
      * Create a new NonAncientEventWindow with the given bounds. The latestConsensusRound must be greater than or equal
      * to the first round of consensus.  If the minimum round non-ancient is set to a number lower than the first round
      * of consensus, the first round of consensus is used instead.  The minGenNonAncient value must be greater than or
      * equal to the first generation for events.
      *
      * @param latestConsensusRound the latest round that has come to consensus
-     * @param minRoundNonAncient   the minimum round that is non-ancient
-     * @param minGenNonAncient     the minimum generation that is non-ancient
-     * @param minGenNonExpired     the minimum generation that is non-expired
-     * @param minRoundNonExpired   the minimum round that is non-expired
+     * @param ancientThreshold     the minimum ancient indicator value for an event to be considered non-ancient
+     * @param expiredThreshold     the minimum ancient indicator value for an event to be considered not expired
```

Comment:
Yes, this is intentional. Our terminology currently sucks since we need a term that describes either generation or birth round. The term we settled on was `ancient indicator` when you want to talk about whatever thing currently is used to decide if an event is ancient.